staff/fernando-costa/bash/README.md

```diff
@@ -0,0 +1,119 @@
+# bash
+Command-line interface commands typical in BASH.
+
+## pwd
+pwd= path work directory => which folder we are now
+```bash
+$ pwd
+```
+
+## ls
+all the folders
+```bash
+$ ls
+```
+
+## ls -l
+all folders we have, but with more info
+```bash
+$ ls -l
+```
+
+## touch    
+it creats a file i.e. hello.world
+```bash
+$ touch
+```
+
+## ls -a
+hidden folders
+```bash
+$ ls -a
+```
+
+## mkdir (make directory)
+it creats a new directory
+```bash
+$ mkdir
+```
+
+## cd
+it changes a directory
+```bash
+$ cd
+```
+
+## mv
+move  i.e $ mv hello.world hola.mundo
+```bash
+$ mv
+```
+
+## clear
+```bash
+$ clear
+```
+
+## kill -9
+```bash
+$ kill
+```
+
+## tree 
+see the path fo the files
+```bash
+```
+## rm -rf 
+remove all folders and its contents
+```bash
+$ rm -rf workspace/hello world
+```
+
+# git branch -D
```

Comment:
Git commands should not be included in this file. If you want, you can create another branch with your git-cheat-sheet file🧐

Reply:
All Git commands have been removed!